github_graphql.go

```diff
@@ -0,0 +1,69 @@
+package resource
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/google/go-github/v32/github"
+	"github.com/shurcooL/githubv4"
+)
+
+func (g *GitHubClient) ListReleases() ([]*github.RepositoryRelease, error) {
+	if g.clientV4 == nil {
```

Comment:
This feels a bit defensive - it should always be non-nil, right?

Reply:
yes, should be non nil, I added this check just to have more safety there